### ISSUE TITLE
Controllers

### ISSUE THREAD

**Issue by sandism** [Xethron/finance#1]
Hi. I suppose there are no controller files ready yet, just database structure?

**Comment by Xethron** [comment]
Hello Sandis,

Unfortunately yes. Got a bit distracted. Will probably start working on it again soon. You looking to build something similar?

**Comment by sandism** [comment]
@Xethron Yes,  also interested in creating something similar, in fact especially because of idea importing bank statements/transaction history. I'm too lazy to write everything by myself. Many available finance management sites/systems out there are limited to features  for free account. Why to pay someone if  I can build my own. Well but also problem occurs that I don't have enough time to code everything from 0 as I want so that's why started to looking for something already done here.

**Comment by Xethron** [comment]
Exactly why I started with this. I want something that I can customize myself. You are welcome to have a look at the little I have done, and we can work on this together if you want.
I wrote an import tool, that imports and maps everything in the DB. And then I mostly just ran SQL queries against it to get the results I wanted. So next step would be to move those into an interface of sorts.
